### PR TITLE
Create SUPPORT.md

### PR DESCRIPTION
**Pull request by asears**
https://docs.github.com/en/github/building-a-strong-community/adding-support-resources-to-your-project

## 1. General summary of the pull request

Improves #1670

### CODE REVIEW

**Comment by bravo-kernel** [comment]
I support adding a `SUPPORT.md` file but think the file itself should only contain a link to the correlation support page at the https://pester.dev website.

This way we only need to maintain a single point of truth.

**Comment by bravo-kernel** [comment]
Thanks for the update @asears but I would also remove the Questions part. 

FWIW I realize there is no "Support" section on the pester.dev website yet but that one can easily be created (using your suggested content).

**Comment by bravo-kernel** [comment]
Apologies, let me rephrase... perhaps we can create a place for your suggestion on the website. Perhaps even renaming the current "Contributing" label to "Support".

image


```
## Questions
[Pester v5](https://github.com/pester/Pester/issues/1218), [tweet/direct message to nohwnd](https://twitter.com/nohwnd), [#testing](https://powershell.slack.com/messages/C03QKTUCS/)
```

**Comment by asears** [comment]
Removed the questions section of support if it's redundant with the information on the docs site.  

From Github blog:

```plaintext
Instead of describing how to contribute to the project like CONTRIBUTING files do, SUPPORT files can be used to direct users to dedicated support resources, such as community forums, FAQ documents, or corporate support channels.

For more information, see the SUPPORT file documentation, and of course, you can always contact GitHub Support if you have any questions.
```

I think that contributing heading probably makes sense from above standpoint.

Adding another section for support if required?

**Comment by bravo-kernel** [comment]
@nohwnd this looks good to me

**Comment by nohwnd** [comment]
Thanks!
